react/src/base/inputs/HugeSelect.stories.js

```diff
@@ -69,3 +66,105 @@ hugeSelectBox.story = {
     ]
   },
 };
+
+export const invalidHugeSelectBox = () => (
+  <SprkSelectionInput
+    label="Select Box Label"
+    choices={[
+      {
+        label: 'Option 1',
+        value: 'option-1',
+      },
+      {
+        label: 'Option 2',
+        value: 'option-2',
+      },
+      {
+        label: 'Option 3',
+        value: 'option-3',
+      },
+      {
+        label: 'Grouped Options',
+        options: [
+          {
+            label: 'Grouped Option 1',
+            value: 'grouped-option-1',
+          },
+          {
+            label: 'Grouped Option 2',
+            value: 'grouped-option-2',
+          },
+          {
+            label: 'Grouped Option 3',
+            value: 'grouped-option-3',
+          },
+        ],
+      },
+    ]}
+    variant="hugeSelect"
+    name="select"
+    valid={false}
+    errorMessage="There is an error in the field."
+    defaultValue=""
+  />
+);
+
+invalidHugeSelectBox.story = {
+  name: 'Invalid',
+  parameters: {
+    jest: [
+      'SprkSelectionInput',
+    ]
+  },
+};
+
+export const disabledHugeSelectBox = () => (
+  <SprkSelectionInput
+    label="Select Box Label"
+    choices={[
+      {
+        label: 'Option 1',
+        value: 'option-1',
+      },
+      {
+        label: 'Option 2',
+        value: 'option-2',
+      },
+      {
+        label: 'Option 3',
+        value: 'option-3',
+      },
+      {
+        label: 'Grouped Options',
+        options: [
+          {
+            label: 'Grouped Option 1',
+            value: 'grouped-option-1',
+          },
+          {
+            label: 'Grouped Option 2',
+            value: 'grouped-option-2',
+          },
+          {
+            label: 'Grouped Option 3',
+            value: 'grouped-option-3',
+          },
+        ],
+      },
+    ]}
+    variant="hugeSelect"
+    name="select"
+    disabled={true}
```

Comment:
noticed that some of these in here have disabled and some have disabled={true}. we could standardize on one way